autogen/versions.tf

```diff
@@ -16,4 +16,8 @@
 
 terraform {
   required_version = ">= 0.12"
+  required_providers {
```

Comment:
Maybe here missed `=`

Reply:
According to [documentation](https://www.terraform.io/docs/configuration/terraform.html#specifying-required-provider-versions) - no.